source/administration/bulk-export-data.rst

```diff
@@ -310,3 +445,55 @@ Reply object
       <td>The timestamp for the reply, in milliseconds since the Unix epoch.</td>
     </tr>
   </table>
+  
+Reaction object
+---------------
+
+This object is a member of the Post object.
+
+.. raw:: html
+
+  <table width="100%" border="1" cellpadding="5px" style="margin-bottom:20px;">
+    <tr class="row-odd">
+      <th class="head">Field name</th>
+      <th class="head">Type</th>
+      <th class="head">Description</th>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">user</td>
+      <td valign="middle">string</td>
+      <td>The username of the user for this reply.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">emoji_name</td>
+      <td valign="middle">string</td>
+      <td>The emoji of the reaction.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">create_at</td>
+      <td valign="middle">int</td>
+      <td>The timestamp for the reply, in milliseconds since the Unix epoch.</td>
+    </tr>
+  </table>
+
+Emoji object
```

Comment:
Maybe the Custom Emoji must be referenced in other places in the documentation, like in the global reference of objects that can be exported/imported.

Reply:
This is already included in the bulk import docs.  Added the objects in a list view to the top of this document as well to be consistent.